src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -12,38 +13,36 @@
 
 namespace Dynamo.Wpf.Utilities
 {
-    internal class CrashReportArgs : EventArgs
+    /// <summary>
+    /// Event argument for CER (crash error reporting) tool.
+    /// It contains options on what to send out with the crash report.
+    /// </summary>
+    public sealed class CrashErrorReportArgs : CrashPromptArgs
     {
-        public bool SendLogFile = true;
-        public bool SendSettingsFile = true;
-        public bool SendDynFile = true;
-        public bool SendRecordedCommands = true;
-
-        internal DynamoViewModel viewModel;
-
-        internal CrashReportArgs(DynamoViewModel dynamoViewModel)
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="viewModel"></param>
+        /// <param name="e"></param>
+        public CrashErrorReportArgs(DynamoViewModel viewModel, Exception e) : base(e) 
         {
-            viewModel = dynamoViewModel;
+            this.viewModel = viewModel;
         }
+
+        public bool SendLogFile { get; set; } = true;
+        public bool SendSettingsFile { get; set; } = true;
+        public bool SendDynFile { get; set; } = true;
+        public bool SendRecordedCommands { get; set; } = true;
```

Comment:
can we collect all these by default with CER ? 
Is this more of a legal question ? I ca bring this up
FYI @Amoursol

Reply:
@pinzart we will need to check with Legal about any of these things (Unless it was correct as per Standup yesterday with Jackie). Assume that if users opt into CER, that these will be covered.